.github/workflows/containerize.yml

```diff
@@ -31,3 +31,16 @@ jobs:
 
     - name: Push Container
       run: RELEASE_BUILD=true make push-agent-image push-agentctl-image
+  build-container-windows:
+    name: Build Windows Container
+    runs-on: windows-2019
+    steps:
+    - name: Login to Docker Hub
+      run: docker login -u '${{ secrets.DOCKER_USER }}' -p '${{ secrets.DOCKER_PASS }}'
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Build and Push Container 
+      run: |
+        $sha=(git rev-parse --short=7 HEAD)
+        docker build . -f .\cmd\agent\Dockerfile.windows -t grafana/agent:nano-1809-$sha
```

Comment:
Dumb question: is the tag name temporary or does it need to be prefixed with `nano-1809` for some reason?

Reply:
it would be better if we could figure out how to do a multi-arch image e.g windows and linux. I think @shorez might this better than me.

Reply:
I'm using buildx in #161 which will be merged shortly but I think the Dockerfile here is different enough that that wouldn't work for a multiarch build